activerecord/test/cases/transactions_test.rb

```diff
@@ -77,7 +77,7 @@ def test_rollback_dirty_changes_even_with_raise_during_rollback_doesnt_commit_tr
 
       assert_equal "The Fifth Topic of the day", topic.reload.title
     ensure
-      ActiveRecord::Base.clear_all_connections!
+      ActiveRecord::Base.clear_all_connections!(:all)
```

Comment:
So in most cases we didn't need to update all of these but sometimes a reading connection is left behind and these will be flaky. It was easier to just pass `:all` to every call for these. I spent some time trying to ensure connections are always entirely cleaned up but it created more headaches then it was worth.